static/monorail-2.0.yaml

```diff
@@ -4680,6 +4680,39 @@ paths:
       - Write
       - workflowsModify
     x-swagger-router-controller: workflows
+  /notification/alerts:
```

Comment:
@yyscamper @anhou We've discussed this in the past.  We will use privileges and firewall rules to separate southbound and northbound routes.  In this case the Dell server uses https, not http, so even without further refactoring, it cannot use the southbound router.  It needs to use port 8443.  This is an entirely seperate discussion that is outside the scope of this PR.